project-base/app/src/Migrations/Version20240722095556.php

```diff
@@ -0,0 +1,29 @@
+<?php
+
+declare(strict_types=1);
+
+namespace App\Migrations;
+
+use Doctrine\DBAL\Schema\Schema;
+use Shopsys\MigrationBundle\Component\Doctrine\Migrations\AbstractMigration;
+
+class Version20240722095556 extends AbstractMigration
+{
+    /**
+     * @param \Doctrine\DBAL\Schema\Schema $schema
+     */
+    public function up(Schema $schema): void
+    {
+        $this->sql('ALTER TABLE stores RENAME COLUMN locationLatitude to latitude');
```

Comment:
it should be `location_latitude`, and `location_longitude` (the DB column names are actually "translated" to underscore from the camelCaps that is used in the entities)

Reply:
Thanks, adjusted.